control/users/migrations/0009_auto_20181018_1629.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
```

Comment:
Be careful with the migrations, since they can collide with migrations in other branchs/PRs